custom-prom.md

```diff
@@ -20,16 +35,19 @@ Kubecost requires the following minimum versions:
 - cAdvisor - kubelet v1.11.0+ (May 18)
 - node-exporter - v0.16+ (May 18) [Optional]
 
-## Implementation Steps
-
-1. Pass the following parameters in your helm [values file](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml):
+## Steps to disable Kubecost's Prometheus Deployment
 
-   * `global.prometheus.fqdn` to match your local Prometheus service address with this format ` http://<prometheus-server-service-name>.<prometheus-server-namespace>.svc`
-   * `global.prometheus.enabled` set to `false`
+**Before contintuing, see the note above about Kubecost's bundled prometheus**
 
-    Pass this updated file to the Kubecost helm install command with `--values values.yaml`
+1. Pass the following parameters in your helm install:
 
-    Or add `--set global.prometheus.fqdn=http://<prometheus-server-service-name>.<prometheus-server-namespace>.svc --set global.prometheus.enabled=false` the end of your helm install command
+  ```sh
+  helm upgrade --install kubecost \
+    --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \
+    --namespace kubecost --create-namespace \
+    --set global.prometheus.fqdn=http://<prometheus-server-service-name>.<prometheus-server-namespace>.svc:port \
```

Comment:
If customer uses hosted prometheus instead of local prometheus installation, it should be <https://$prometheus-query-endpoint>

Reply:
On my deployment, it is both http and in another namespace
Port required if not 80.

no?

Reply:
For hosted deployment like Grafana cloud, port is not required and it can be https. Eg: 
```
helm upgrade -i kubecost kubecost/cost-analyzer \
    --namespace kubecost \
    --set global.prometheus.fqdn=https://prometheus-prod-10-prod-us-central-0.grafana.net/api/prom/ \
    --set global.prometheus.enabled=false
```

Reply:
good point!